src/components/LibrariesListItem.tsx

```diff
@@ -54,7 +54,8 @@ export default class LibrariesListItem extends React.Component<LibrariesListItem
   render(): JSX.Element {
     let style = this.state.color;
     let { name, short_name } = this.props.library.basic_info;
-    let headerText = `${name} (${short_name})`;
+    let plsID = this.props.library.basic_info.pls_id ? `: ${this.props.library.basic_info.pls_id}` : "";
```

Comment:
Is this information important enough to view in the header with the name of the library? It’s confusing enough to see the name and then the short name (a random string of chars) in parenthesis in the header. Then this ID would be there without any label. I don't think having this here helps.

Reply:
I do think it's important enough to view in the header, unless we implement a way to search for it (which would be a separate ticket).  But I see what you mean about it being confusing; I'll make it clearer.

Reply:
I thought that id was used about once every six months? It doesn't seem that useful. I figured it'd go with the other key value library information pairs in a table.

Reply:
Ok.  I was thinking more because we're going to be inputting all of the PLS IDs, so it would be good to be able to see which libraries don't yet have one without having to open all the panels.  But...what would you think about putting something in the header to indicate if a library DOESN'T have one?  Or (in a different branch) putting in a filter (just on the front end) to show just the libraries that don't have one?

Reply:
The filter option sounds very good because you can extend it to filter for other library attributes.